src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.Strings.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_Strings
+    {
+        private const int DataSize = 100_000;
+
+        private ArrayBufferWriter<byte> _arrayBufferWriter;
+
+        private string[] _stringArrayValues;
+        private byte[][] _stringArrayValuesUtf8;
+
+        [Params(true, false)]
+        public bool Formatted;
+
+        [Params(true, false)]
+        public bool SkipValidation;
+
+        public enum Escape
+        {
+            AllEscaped,
+            OneEscaped,
+            NoneEscaped
+        }
+
+        [Params(Escape.AllEscaped, Escape.OneEscaped, Escape.NoneEscaped)]
+        public Escape Escaped;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _arrayBufferWriter = new ArrayBufferWriter<byte>();
+
+            _stringArrayValues = new string[DataSize];
+            _stringArrayValuesUtf8 = new byte[DataSize][];
+
+            for (int i = 0; i < DataSize; i++)
+            {
+                _stringArrayValues[i] = GetString(5, 100, Escaped);
```

Comment:
Is it interesting to have different buckets of tests based on string length? One imagines we could collect some data about this and know that commonly strings in JSON payloads are 0-7, 7-25, etc. Having this differentated a bit would also cover the hello world case better.

Reply:
> One imagines we could collect some data about this and know that commonly strings in JSON payloads are 0-7, 7-25, etc. 

That would be great and really useful, especially for some of the E2E perf tests I plan to add. However, it would require some time for me to deliberately collect the json payload data for this purpose. Generally, size of JSON **property** strings can be reasoned about (and mostly are 3-10 characters). The JSON **value** strings are quite an open bag.

I would like to defer this to later.